staging/src/k8s.io/apiserver/pkg/storage/storagebackend/factory/etcd3_test.go

```diff
@@ -46,3 +53,28 @@ func Test_atomicLastError(t *testing.T) {
 		t.Fatalf("Expected: \"now error\" got: %s", err.Error())
 	}
 }
+
+func TestClientWithGrpcHealthcheck(t *testing.T) {
+	codec := apitesting.TestCodec(codecs, examplev1.SchemeGroupVersion)
+
+	etcdConfig := testserver.NewTestConfig(t)
+
+	client := testserver.RunEtcd(t, etcdConfig)
+	cfg := storagebackend.Config{
+		Type: storagebackend.StorageTypeETCD3,
```

Comment:
Probably we could ask etcd team to publish fail point enabled etcd binaries for kube-apiserver to trigger the defrag latency in the test environment. 

Otherwise, the testing won't capture the above missing import issues.

Reply:
Ping @serathius above ^^